emqx/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "fa40ec7e-e8f6-4c4b-a675-31716b23a9df",
+  "app_id": "emqx",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Emqx integration with openmetrics v2",
+    "title": "EMQX",
+    "media": [],
```

Comment:
Please add at least one image. Usually this image is a screenshot of your dashboard or some kind of visual representation of the data being sent in